script/Ciel/Day 10/10_01_CIEL10_5B.txt

```diff
@@ -296,27 +318,27 @@ Senpai?!
 [sha:bc620dadbd13115f88b4327647d216c883bc1727]{
 -- Page 375, Offset 28088.
 -- 「―――確かめに行きます。
-"I'm going there to ascertain this.
+"I need to go verify this.
 }
 [sha:13f564a97af6df2e393296e9fcaf23f9e7ad1f5a]{
 -- Page 375, Offset 28089.
 -- 遠野くんはここに残っていてください」
-Please stay right here, Tohno-kun."
+Please don't leave this room, Tohno-kun."
 }
 [sha:d1d8d6825f5c028607dd143d32e82ff78cbb1edd]{
 -- Page 376, Offset 28090.
 -- 先輩は出口の扉に手をかける。
-Senpai puts her hand on the exit door.
+Senpai puts her hand on the door to leave.
 }
 [sha:270c682dfee054ac68b044a817c8a92d56b82389]{
 -- Page 376, Offset 28091.
 -- その背中は悲壮な決意に満ちていた。
-She was filled with a heroic but tragic determination.
+She seems to be filled with a grim determination.
 }
 [sha:c16991f5ef85602ef1233ffca223c9c3e95b019d]{
 -- Page 376, Offset 28092.
 -- ……突っ立っている場合じゃない。
-...This isn't the time to be standing around!
+I can't just stand here!
```

Comment:
I'd like there to some kind of thing after 'here', just so that there is more of a qualifier
>I can't just stand here idling away!

>I can't just stand here doing nothing!